src/status_im/communities/core.cljs

```diff
@@ -110,15 +111,29 @@
 (rf/defn left
   {:events [::left]}
   [cofx response-js]
-  (rf/merge cofx
-            (handle-response cofx response-js)
-            (navigation/pop-to-root-tab :chat-stack)
-            (activity-center/notifications-fetch-unread-count)))
+  (let [community-name (aget response-js "communities" 0 "name")]
+    (rf/merge cofx
+              {:dispatch-n [[:sanitize-messages-and-process-response response-js]
+                            [:toasts/create
+                             {:icon       :placeholder
+                              :icon-color "green"
+                              :text       (i18n/label :t/left-community {:community community-name})}]]}
+              (navigation/pop-to-root-tab :chat-stack)
+              (activity-center/notifications-fetch-unread-count))))
 
 (rf/defn joined
   {:events [::joined ::requested-to-join]}
   [cofx response-js]
-  (handle-response cofx response-js))
+  (let [[event-name _] (:event cofx)
+        community-name (aget response-js "communities" 0 "name")]
+    {:dispatch-n [[:sanitize-messages-and-process-response response-js]
+                  [:toasts/create
+                   {:icon       :placeholder
+                    :icon-color "green"
```

Comment:
should this color come from quo2/foundations ?

Reply:
yes color should be a property in the component not the color value